Aptoide/aptoide-client-v8
# [ASV-989] App of the Week transition

**What does this PR do?**

 Adds a transition between the bottom app card and the placeholder app card when you scroll in the editorial view

**Database changed?**

   No

**Where should the reviewer start?**

- EditorialFragment.java

**How should this be manually tested?**

Rotate Screen
Scroll back and forth
Go to other screen and come back and see if everything is working how it’s suppose
Do several download flows
Just try to break it 
To test it in different positions, replace the type from normal to app_placeholder, where you want and put the other normal in charles

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/ASV-989

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.) 




**Code Review Checklist**
